src/material/tabs/tab-group.ts

```diff
@@ -79,6 +79,7 @@ const ENABLE_BACKGROUND_INPUT = true;
     '[class.mat-mdc-tab-group-dynamic-height]': 'dynamicHeight',
     '[class.mat-mdc-tab-group-inverted-header]': 'headerPosition === "below"',
     '[class.mat-mdc-tab-group-stretch-tabs]': 'stretchTabs',
+    '[attr.mat-align-tabs]': 'alignTabs',
```

Comment:
This will likely break existing places where users have set the `mat-align-tabs` attribute themselves. To handle it properly we would have to introduce a `mat-align-tabs` input and toggle a class here instead.

Reply:
added a test case to verify that `mat-align-tabs` input takes preceding over config, lemme know if this is the change you proposed.